### ISSUE TITLE
Unselectable mixin

### ISSUE THREAD

**Issue by lexer** [Compass/compass#417]
User select is not cross browser. It is reasonable to include following mixin to compass.

@mixin unselectable {
   -moz-user-select: -moz-none;
   -khtml-user-select: none;
   -webkit-user-select: none;
   -o-user-select: none;
   user-select: none;
}

**Comment by mirisuzanne** [comment]
This should actually be:

```
@mixin user-select($select) {
  -moz-user-select: -moz-#{$select};
  -khtml-user-select: $select;
  -webkit-user-select: $select;
  -o-user-select: $select;
  user-select: $select;
}
```

or something along those lines. I haven't tested. But the mixin should mimic the CSS property in full, not a single value for that property.

**Comment by chriseppstein** [comment]
@ericam We're you planning on implementing this?

**Comment by mirisuzanne** [comment]
Let's say yes. Sure.

**Comment by mirisuzanne** [comment]
It looks like this has been removed from the W3C recommendation: http://www.w3.org/TR/css3-userint

It was in an old Working Draft:

http://www.w3.org/TR/2000/WD-css3-userint-20000216

But Is not in the Candidate Recommendation:

http://www.w3.org/TR/css3-ui

There is some experimental browser support. How do we handle that?

**Comment by chriseppstein** [comment]
It doesn't really matter if it's a standard or not, if most browsers implement it, then we can add a mixin for it with caveats in the docs.

**Comment by mirisuzanne** [comment]
That's fine, but I can't find it on any support tables that I know of. Has anyone done that testing? If not, I can do some basic testing next week. What do we consider 'most browsers'?

**Comment by adamstac** [comment]
I've used this in Chrome v12.0.742.112 and Firefox v4.0.1 and Safari 5.0.5 on Mac. Though, I haven't tested this on any other browsers.

If the browsers support it, I'd say add it. I was just checking to see if we had already and Wynn pointed me here to share my usage and comments. What Eric showed off above seems like a nice mixin to support this.

**Comment by mirisuzanne** [comment]
closed by 92981fb5d3
